neurom/core/morphology.py

```diff
@@ -240,23 +271,44 @@ def iter_neurites(obj, mapfun=None, filt=None, neurite_order=NeuriteIter.FileOrd
         >>> mapping = lambda n : len(n.points)
         >>> n_points = [n for n in iter_neurites(pop, mapping, filter)]
     """
-    neurites = ((obj,) if isinstance(obj, Neurite) else
-                obj.neurites if hasattr(obj, 'neurites') else obj)
+    neurites = (
+        (obj,)
+        if isinstance(obj, Neurite)
+        else obj.neurites
+        if hasattr(obj, "neurites")
+        else obj
+    )
+
     if neurite_order == NeuriteIter.NRN:
         if isinstance(obj, Population):
             warnings.warn('`iter_neurites` with `neurite_order` over Population orders neurites'
                           'within the whole population, not within each morphology separately.')
         last_position = max(NRN_ORDER.values()) + 1
         neurites = sorted(neurites, key=lambda neurite: NRN_ORDER.get(neurite.type, last_position))
 
+    if use_subtrees:
+
```

Comment:
nit: let's start dropping the extra line between `if` and the next statement, to keep it the same as other code

Reply:
I will try :D